openerp/tools/convert.py

```diff
@@ -199,13 +199,17 @@ def _process(s):
 class xml_import(object):
 
     @staticmethod
+    def str2bool(value):
+        return value.lower() not in ('0', 'false', 'off')
+
+    @staticmethod
     def nodeattr2bool(node, attr, default=False):
         if not node.get(attr):
             return default
         val = node.get(attr).strip()
         if not val:
             return default
-        return val.lower() not in ('0', 'false', 'off')
+        return self.str2bool(val)
```

Comment:
I using `self` in `staticmethod` hahaha sorry!
Fixed creating a new global method like as `_eval_xml`...